modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -116,40 +144,22 @@ The command also completes the following actions:
 +
 [IMPORTANT]
 ====
-The Agent-based Installer uses {op-system-first}. The rootfs image, which is mentioned in a later listed item,  is required for booting, recovering, and repairing your operating system.
+The Agent-based Installer uses {op-system-first}. The rootfs image, which is mentioned in a later step, is required for booting, recovering, and repairing your operating system.
 ====
 
-. Configure the `agent-config.yaml` configuration file to meet your organization’s requirements.
-+
-.Example `agent-config.yaml` configuration file that sets values for an IPv4 formatted network.
-[source,yaml]
-----
-apiVersion: v1alpha1
-metadata:
-  name: <cluster_name> // <1>
-  namespace: <cluster_namespace> <2>
-rendezvousIP: <ip_address_from_CIDR> <3>
-bootArtifactsBaseURL: <server_URL> <4>
-# ...
-----
-<1> The cluster name that you specified in your DNS record.
-<2> The namespace of your cluster on {product-title}.
-<3> If you use IPv4 as the network IP address format, ensure that you set the `rendezvousIP` parameter to an IPv4 address that the VCN’s Classless Inter-Domain Routing (CIDR) method allocates on your network. Also ensure that at least one instance from the pool of instances that you booted with the ISO matches the IP address value you set for `rendezvousIP`.
-<4> The URL of the server where you want to upload the rootfs image.
+. Disconnected environments only: Upload the rootfs image to a web server.
 
-. Apply one of the following two updates to your `agent-config.yaml` configuration file:
-+
-* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the rootfs image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload rootfs to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
+..  Go to the `./<installation_directory>/boot-artifacts` directory that was generated when you created the minimal ISO image.
+
+.. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload the rootfs image to the location specified in the `bootArtifactsBaseURL` parameter of the `agent-config.yaml` file.
 +
 For example, if the `bootArtifactsBaseURL` parameter states `\http://192.168.122.20`, you would upload the generated rootfs image to this location, so that the Agent-based installer can access the image from `\http://192.168.122.20/agent.x86_64-rootfs.img`. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the rootfs image from the `\http://192.168.122.20/agent.x86_64-rootfs.img` location into the system memory.
 +
 [NOTE]
```

Comment:
_If possible_, try to avoid stacking admonitions.

Reply:
In a later effort I will consider whether this Note text even needs to be in a note rather than in a paragraph